aten/src/ATen/native/quantized/cpu/qnnpack/src/conv-run.cc

```diff
@@ -281,6 +281,7 @@ struct QnnpackDeleter {
   }
 };
 
+__attribute__((no_sanitize("function")))
```

Comment:
@kimishpatel seems like this is still causing the ASAN issue - https://circleci.com/gh/pytorch/pytorch/5266916?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Sorry, little out of clue at this point. I think one thing can be tried is no_sanitize("undefined"). This should be a big hammer that bypasses UBSAN on this function. If that doesnt work either then probably we are not able to apply the attribute to the function. Best if you can repro and try these things locally.